extensions/datastores/kudu/src/main/java/org/locationtech/geowave/datastore/kudu/operations/KuduWriter.java

```diff
@@ -46,6 +48,13 @@ public synchronized void write(GeoWaveRow row) {
     }
   }
 
+  public void batchedWrited(GeoWaveRow[] rows) {
+    if (batchHandler.setAutoFlushMode()) {
```

Comment:
Should probably do something if this fails (display an error or try to write the rows normally)